ui/pages/notifications/notification-components/stake/stake.tsx

```diff
@@ -148,7 +151,7 @@ export const components: NotificationComponent<StakeNotification> = {
       );
     },
     body: {
-      type: 'body_onchain_notification',
+      type: NotificationComponentType.OnChainBody,
```

Comment:
In the future it would be nice to split this out in a separated PR and do this refactor before mixing in Snaps changes IMO

Reply:
Yeah I suppose, did this on Maarten's request lol